app/src/main/java/lt/vilnius/tvarkau/utils/AnalyticsUtil.kt

```diff
@@ -0,0 +1,35 @@
+package lt.vilnius.tvarkau.utils
+
+import android.app.Activity
+import android.content.Context
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import com.google.firebase.analytics.FirebaseAnalytics
+import lt.vilnius.tvarkau.entity.Problem
+
+object AnalyticsUtil {
```

Comment:
I'd would prefer if we could make it as an implementation.

`inferface Analytics`
`class RemoteAnalytics : Analytics`

and injected using dagger.
That way we could test implementation and integration. What do you think?

That one could write in fragments tests

`verify(analytics).trackProblem(any())`